### PR TITLE
fix bugs about ModelBO's member variable calling

### PR DESCRIPTION
**Pull request by Lionjump0723**
Closes #

#### What has been done to verify that this works as intended?

The main purpose of this pull request is to show my contribution in the last 4 days in detail. 

 I've tried to run the newly added "playground_convert.py" to test the profiler module. Between line 10  and the line 18 is the first part of the code, where I convert pytorch model into torchscript model which the frame can actually serve with. Before running this part, I've changed the weight address in "./example/resnet50.yml" to the address of the post-converting model so that I can use the register_model_from_yaml API to register the post-converting model. 

After that, I've tried to run the second part of the code (from line 19). Through this process I've found that in the manager.py as well as \*client.py, the programers try to call the member variable 'name' of ModelBO, which doesn't exist. I've fixed these bugs, and finally I get stuck at grpc errors. 

#### Why is this the best possible solution? Were any other approaches considered?

...

#### How does this change affect users? Describe intentional changes to behavior and behavior that could have accidentally been affected by code changes. In other words, what are the regression risks?

It has no affects to users. 

#### Does this change require updates to documentation?

No. 

#### Before submitting this PR, please make sure you have:

- [] run `python -m pytest tests/` and confirmed all checks still pass.
- [] verified that any code or assets from external sources are properly credited.

### CODE REVIEW

File: playground_convert.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+from pathlib import Path
```
Comment (HuaizhengZhang) [inline playground_convert.py:1]: Why put this playground file in the root directory

Comment (HuaizhengZhang) [reply playground_convert.py:1]: Its inappropriate.

**Review by HuaizhengZhang — CHANGES_REQUESTED** [review]

File: playground_convert.py (hunk 1 above)
Comment (Lionjump0723) [reply playground_convert.py]: The main purpose of this pull request is to show my process of debugging. The playground file is a kind of test file, but it does not run successfully, so I don't put it in test directory. Now I've deleted the playground file and it's appropriate for merge now.

**Comment by HuaizhengZhang** [comment]
@YuanmingLeee Is this ok?

**Review by YuanmingLeee — APPROVED** [review]
lgtm

**Pull request merged**